cita-chain/types/src/header.rs

```diff
@@ -31,29 +31,28 @@ use time::get_time;
 use super::Bytes;
 pub use crate::block_number::BlockNumber;
 use hashable::{Hashable, HASH_NULL_RLP};
-use util::HeapSizeOf;
 
 lazy_static! {
     pub static ref ZERO_BLOOM: Bloom = Bloom::from([0x00; 256]);
 }
 
 #[derive(Debug, Clone, Eq)]
 pub struct OpenHeader {
-    /// Parent hash.
+    /// Previous block hash.
     parent_hash: H256,
     /// Block timestamp.
     timestamp: u64,
     /// Block number.
     number: BlockNumber,
     /// Transactions root.
     transactions_root: H256,
-    /// Block gas limit.
+    /// Block quota limit.
     quota_limit: U256,
-    /// the proof of the block
+    /// Block proof
     proof: ProtoProof,
-    /// The version of the header.
+    /// Block version.
```

Comment:
Block version?

Reply:
Is there something wrong?

Reply:
We have `protocol version` and `software version`.
I don't think that `block version` is a good description.